UI/Marker/EnemySetPlacemark.gd

```diff
@@ -43,3 +44,10 @@ func can_drop_data(_position, data):
 func drop_data(_position, data):
 	add_enemy(Enemy.new(data))
 	print_debug("Placed %s at %s (%d %d %d %d) " % [tr(data.name), tr(str("STAGE_NAME_",_enemy_set.stage_id)), _enemy_set.stage_id, _enemy_set.layer_no, _enemy_set.group_id, _enemy_set.subgroup_id])
+
+func _to_hide():
+	self.visible = false
+	
+func _to_unhide():
```

Comment:
Couldn't both this and the code in GatheringSpotPlacemark be moved to the superclass?